source/run-command/tests/unified/runCommand.yml

```diff
@@ -65,9 +65,6 @@ tests:
                 $db: *db
                 lsid: { $$exists: true }
                 $readPreference: { $$exists: false }
-                apiVersion: { $$exists: false }
-                apiStrict: { $$exists: false }
-                apiDeprecationErrors: { $$exists: false }
```

Comment:
Was this removed for compatibility with running the entire test suite with `requireApiVersion: true` (see: [Stable API Tests](https://github.com/mongodb/specifications/blob/master/source/versioned-api/tests/README.rst))?